docs/Syntax.md

```diff
@@ -0,0 +1,82 @@
+# Syntax
+
+Before the start, you need to configure the rules by which the router will decide which of the shards to send each request to.
+
+For this purpose, SPQR has an administrative console. This is an app that works by PostgreSQL protocol and you can connect to it by usual psql. You can find the console port in your config file.
+
+```
+➜  psql "host=localhost sslmode=disable user=news dbname=news port=7432"
+
+SQPR Router Administrator Console
+Here you can set up your own routing rules
+------------------------------------------------
+You can find the documentation here
+https://github.com/pg-sharding/spqr/tree/master/docs
+
+psql (14.5 (Homebrew), server console)
+Type "help" to get help.
+
+news=> SHOW segments;
+list of data segments
+---------------------------
+data card with the shard01 identifier data card
+with the shard02 identifier
+(2 rows)
+```
+
+To make all things work, the router needs to know the following:
+
+- What tables are you query
+- Which columns in each table should the router look
+- The type of these columns
+- Mapping from [range of values] to [shard number]
+
+Let's create a distribution first:
```

Comment:
Maybe it's better to describe various sharidng-related terms before showing an example?